packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1183,6 +1336,153 @@ export class Combobox
     });
   }
 
+  renderAllSelectedIndicatorChip(): VNode {
+    const {
+      compactDisplayMode,
+      isAllSelected,
+      scale,
+      selectedVisibleChipsCount,
+      setAllSelectedIndicatorChipEl,
+    } = this;
+    const label = this.messages.allSelected || "All selected";
```

Comment:
Let's remove the fallback since we've already got messages set up.

Reply:
I just now realized I should probably have a message for `${num} selected` too.

Reply:
Done.  Added one for `selected` as well.